docs/plugin-maintainers-guide.md

```diff
@@ -47,7 +50,59 @@ If you are no longer maintaining a plugin, please take the following steps to fo
 
 3. If you are the last remaining CODEOWNER:
    - The `@backstage/community-plugins-maintainers` group will provide best-effort support for issues and maintenance. Without a dedicated owner, plugin updates and support may be slower or limited.
-   - To help with this, opt into automatic version bump PRs by creating an empty `.auto-version-bump` file in the plugin’s workspace (i.e., `workspaces/${WORKSPACE}/.auto-version-bump`).
+   - To help with this, opt into automatic version bump PRs by creating an empty `.auto-version-bump` file in the plugin's workspace (i.e., `workspaces/${WORKSPACE}/.auto-version-bump`).
+
+## Archiving a Plugin
+
+When a plugin is no longer maintained, has security vulnerabilities, or is being deprecated in favor of alternatives, it should be archived rather than abandoned. The archival process ensures that users are properly notified through npm deprecation warnings while preserving the code for historical reference through Git tags.
```

Comment:
```suggestion
When a plugin is no longer maintained, it should be archived rather than abandoned. The archival process ensures that users are properly notified through npm deprecation warnings while preserving the code for historical reference through Git tags.
```